src/util/locking/RedisLocker.ts

```diff
@@ -172,24 +177,36 @@ export class RedisLocker implements ReadWriteLocker, ResourceLocker, Finalizable
     );
   }
 
-  /* Finalizer methods */
+  /* Initializer & Finalizer methods */
+
+  public async initialize(): Promise<void> {
+    // On server start: remove all existing (dangling) locks, so new requests are not blocked.
+    return this.clearLocks();
+  }
 
   public async finalize(): Promise<void> {
-    // This for loop is an extra failsafe,
-    // this extra code won't slow down anything, this function will only be called to shut down in peace
+    this.finalized = true;
```

Comment:
Could perhaps also overwrite a function here instead of setting this parameter, as mentioned above.